autopts/ptsprojects/zephyr/iutctl.py

```diff
@@ -205,6 +205,7 @@ def rtt_logger_start(self):
 
     def rtt_logger_stop(self):
         if self.rtt_logger:
+            time.sleep(0.1)
```

Comment:
why is this needed?

Reply:
In the case where an issue in the host has caused an operation to fail and in turn a test-case to fail, the rtt logger will often not have the time to pick up an error-print from the IUT before being closed. When this happens, the IUT logs will be incomplete, since any hints to the actual reason of the failure will be missing. This delay is to ensure enough time to prevent such a situation.

Reply:
OK, I guess short comment in code would be good to not forget about this :)

Reply:
Comment added

Reply:
btw with assert message it seems that 0.1 is not enough